rainier-core/src/main/scala/com/stripe/rainier/core/Categorical.scala

```diff
@@ -90,6 +90,11 @@ object Categorical {
       Real(l.size)
     })
 
+  def fromSet[T](ts: Set[T]): Categorical[T] =
```

Comment:
Any reason to prefer this over something like 
```
val p = Real.one / ts.size
ts.map{t => t -> p}.toMap
```
?

Reply:
nope, just wanted to reuse normalize vs "re-implementing", even though it's easy. This is good, I'll fix!

Reply:
Ah, I did use the `foldLeft` to avoid two passes over the set, as `toMap` would have done (I think). I'm keeping that, but I took your `val p` suggestion.